### PR TITLE
Stabilize 1.5

### PR DESCRIPTION
**Pull request by phaseinducer**

### CODE REVIEW

**Comment by damien-carcel** [comment]
GTM

File: features/product/filtering/filter_products_per_with_multiple_metrics.feature [hunk 1]
```diff
@@ -28,7 +28,8 @@ Feature: Filter products with multiples metrics filters
     And I am logged in as "Mary"
     And I am on the products page
 
-  @unstable # TODO Do not merge into the master
+  # TODO Do not merge into the master
```
Comment (jjanvier) [inline features/product/filtering/filter_products_per_with_multiple_metrics.feature]: do not merge into master ;)

File: features/product/filtering/filter_products_per_with_multiple_multiselect.feature [hunk 2]
```diff
@@ -30,6 +30,7 @@ Feature: Filter products with multiples multiselect filters
     And I am logged in as "Mary"
     And I am on the products page
 
+  @unstable
```
Comment (jjanvier) [inline features/product/filtering/filter_products_per_with_multiple_multiselect.feature]: do not merge that neither no?

**Comment by jjanvier** [comment]
gtm once fixed

**Pull request merged**